ui/tests/unit/adapters/secret-engine-test.js

```diff
@@ -62,4 +62,68 @@ module('Unit | Adapter | secret engine', function (hooks) {
     const response = adapter.findRecord(storeStub, 'aws', { path: 'aws' }, snapshot);
     assert.propEqual(response, { data: {} }, 'returns empty data object');
   });
+
+  module('WIF secret engines', function (hooks) {
+    hooks.beforeEach(function () {
+      this.version = this.owner.lookup('service:version');
```

Comment:
I don't think `this.version` and setting to enterprise are necessary for these tests? Is there any version logic in the adapter specifically?